pkg/minishift/addon/parser/addon_parser.go

```diff
@@ -180,12 +180,19 @@ func (parser *AddOnParser) parseCommands(scanner *bufio.Scanner) ([]command.Comm
 }
 
 func createMetaData(header []string) (addon.AddOnMeta, error) {
-	regex, _ := regexp.Compile(`^# ?(.*):(.*)`)
+	regex, _ := regexp.Compile(`^# ?([a-zA-Z-]*):(.*)`)
 	metaMap := make(map[string]string)
+	var key string
 	for _, line := range header {
 		matches := regex.FindAllStringSubmatch(line, -1)
+		if len(matches) == 0 && key != "" {
+			line = strings.TrimPrefix(line, commentChar)
+			metaMap[key] = metaMap[key] + " " + strings.TrimSpace(line)
```

Comment:
This really removes the new line. I think we should keep it. I added for example two lines to the anyuid add-on:

        # Name: anyuid
        # Description: Changes the default security context constraints to allow pods to run as any
        # foo
        # bar

When I then run `minishift addon list --verbose` I get:

    Name       : anyuid
    Description: Changes the default security context constraints to allow pods to run as any user. foo bar
    Enabled    : enabled
    Priority   : 0

I would expect the newlines to be preserved. I always had in mind to add:

    $ minishift add-on describe <name> 

which just dumps the description to the console. An add-on would become self describing of sorts and I would expect that one would less accompanying READMEs for add-ons. One would put the essential parts into the add-on description. 

@jorgemoralespou, I would expect that you also expected the description to preserve the new lines, right?